subsys/llext/llext.c

```diff
@@ -776,6 +777,7 @@ static int llext_link(struct llext_loader *ldr, struct llext *ext, bool do_local
 				LOG_INF("found section symbol %s addr 0x%lx", name, link_addr);
 			} else {
 				/* Nothing to relocate here */
+				LOG_INF("not relocated");
```

Comment:
I'd say this should likely be a LOG_DBG

Reply:
> The base ztest suite should be the same, and run through all the available test extensions like the simplest hello world printk, this sem give/take with a global, and so on.

@teburd I've put one up, have a look and see if you like it / let me know what other test cases you'd like to see.